### PR TITLE
Add daily build and cleanup triggers

### PR DESCRIPTION
**Pull request by LorbusChris**
Easiest to review commit by commit

### CODE REVIEW

**Comment by vrutkovs** [comment]
Not sure if we need a cleanup task - Tekton operator can be [configured to do the pruning](https://docs.openshift.com/container-platform/4.12/cicd/pipelines/automatic-pruning-taskrun-pipelinerun.html) - can we make use of that?

**Comment by LorbusChris** [comment]
We're actually doing it this way already, this change just replaces the current cronjob.
I'm open to investigating the use of the operator further later.

File: base/triggers.tekton.dev/triggertemplates/tekton-cleanup.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      generateName: cleanup-$(tt.params.namespace)-taskrun-
+    spec:
+      serviceAccountName: pipeline
+      taskSpec:
+        params:
+        - name: keep
+        - name: namespace
+        steps:
+        - name: cleanup-pr-tr
+          image: gcr.io/tekton-releases/dogfooding/tkn
```
Comment (lmzuccarelli) [inline base/triggers.tekton.dev/triggertemplates/tekton-cleanup.yaml:27]: @LorbusChris Nice - this is a really neat solution (dogfooding) !!!

**Review by lmzuccarelli — COMMENTED** [review]
I'm going to verify on kind
Other that the verification this is great work @LorbusChris 
/lgtm

**Comment by LorbusChris** [comment]
Merging as this is what's deployed now.

**Pull request merged**